specification.md

```diff
@@ -62,14 +62,57 @@ Each **Span** encapsulates the following state:
   of any type. Not all OpenTracing implementations must support every value
   type.
 - A **SpanContext** (see below)
-- **References** to zero or more causally-related **Spans** (via the
+- [**References**](#references) to zero or more causally-related **Spans** (via the
   **SpanContext** of those related **Spans**)
 
 Each **SpanContext** encapsulates the following state:
 
 - Any OpenTracing-implementation-dependent state (for example, trace and span ids) needed to refer to a distinct **Span** across a process boundary
 - **Baggage Items**, which are just key:value pairs that cross process boundaries
 
+<span id="references"></span>
+
+### References between Spans
+
+A Span may reference zero or more other **SpanContexts** that are causally related. OpenTracing presently defines two types of references: `ChildOf` and `FollowsFrom`. **Both reference types specifically model direct causal relationships between a child Span and a parent Span.** In the future, OpenTracing may also support reference types for Spans with non-causal relationships (e.g., Spans that are batched together, Spans that are stuck in the same queue, etc).
```

Comment:
I reformatted some things, but aside from changing "Span" to "SpanContext" in a few places this copy is unchanged from the docs site.